deploy/secrets.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster1-secrets
+type: Opaque
+data:
```

Comment:
please use `stringData` as in https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/secrets.yaml

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/20/commits/b1ce183403382348f9cce75920e52b7ef593512c